frontend/src/general/dashboard/__tests__/DashboardButton.test.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import Enzyme from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import { describe, expect, it } from "@jest/globals";
+import { Link } from "@material-ui/core";
+
+import DashboardButton from "../DashboardButton";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("<DashboardButton/> Component", () => {
+  let wrapper;
+  const dummyHref = "Title";
+  const dummyBody = "Body";
+
+  describe(`render() with props href:${dummyHref} children:${dummyBody}`, () => {
+    wrapper = Enzyme.shallow(
+      <DashboardButton href={dummyHref}>{dummyBody}</DashboardButton>
+    );
+
+    it("Contains 1 <Link/> component", () => {
+      expect(wrapper.find(Link)).toHaveLength(1);
+      const link = wrapper.find(Link).first();
+      expect(link.props().href).toBe(dummyHref);
+    });
+    it("Contains 1 WithStyles() wrapped Button component", () => {
+      const linkChild = wrapper.find(Link).first().children().first();
+
+      expect(linkChild.type().Naked.displayName).toBe(
```

Comment:
could i ask what `Naked` does/how it works? wasn't able to find much/any documentation about it on my own 😓

Reply:
Naked = not wrapped

Reply:
ahh i see thank uu
`withstyles` wrapper seems like a lot of fun to test 💦💦